my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-info.html

```diff
@@ -43,61 +43,10 @@ <h1>{{ alert.title }}</h1>
   <div layout="row" layout-align="center center" ng-show="profileLoadingState.ephone || profileLoadingState.lcontact || profileLoadingState.einfo">
     <loading-gif data-object="{{[]}}"></loading-gif>
   </div>
+  <p>Students now update their local and emergency contact information in the <a href="/apps/details/student-center">Student Center</a>&nbsp;</p>
```

Comment:
I haven't done web development in a while, but I am thinking this content should go inside a div block - maybe even of the class container__emergency-info like the original div?

Reply:
What work is the `&nbsp;` doing?